homeassistant/components/met/__init__.py

```diff
@@ -33,7 +34,8 @@
     DOMAIN,
 )
 
-URL = "https://aa015h6buqvih86i1.api.met.no/weatherapi/locationforecast/2.0/complete"
+# From https://api.met.no/weatherapi/locationforecast/2.0/documentation
+URL = "https://api.met.no/weatherapi/locationforecast/2.0/complete"
```

Comment:
We cannot change this. As we are a heavy traffic consumer, we have been assigned a different end-point.

Reply:
Yep: https://github.com/home-assistant/core/issues/59554 & https://github.com/home-assistant/core/pull/39583

Can we put a note above it that explains why it is a custom URL?

Reply:
> We cannot change this. As we are a heavy traffic consumer, we have been assigned a different end-point.
> 

Gotha,  but current URL here timeout 99% of the time :-S

Reply:
Sorry for tagging you @geira, could you maybe provide insight on that?

Reply:
The reason Home-Assistant has its own custom subdomain is that the traffic is way above the TOS limit of 20 requests/second, and is currently using about 20% of the total api traffic. For this reason we need to be able to monitor and route traffic to optimize resources. As recently mentioned on our [mailing list](https://lists.met.no/pipermail/api-users/2022-April/000068.html) we are experiencing network problems with slow downloads on the main gateway. Since we have an Easter holiday systems freeze we decided to route the most trafficed subdomains to our standby gateway which previously had been running idle, until we can set up a new redundant pair of servers to balance the load. Please do not use momentary statistics taken during a notified emergency as an excuse to circumvent the TOS.

Reply:
Right, so it is a service disruption, thanks for that valuable information that is appreciated.


> Please do not use momentary statistics taken during a notified emergency as an excuse to circumvent the TOS.

Nobody said we would, please don't approach things from a negative context and throw around TOS policies and such. We are just asking and reaching out. 

Have a great Easter weekend @geira 🥚 🐰 

👍

Reply:
Added a note for the future.

Reply:
That is helpful, thanks @chemelli74 👍